event-protocol/ruby/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Cucumber Limited and contributors
```

Comment:
Never add year to Copyright. It's not required and nobody remembers to update it.